OpenOversight/app/api/v1.py

```diff
@@ -0,0 +1,56 @@
+from http import HTTPMethod
+
+from flask import Blueprint, Response, jsonify
+from sqlalchemy.orm import Query
+
+from OpenOversight.app.models.database import Department
+from OpenOversight.app.utils.flask import limiter
+
+
+v1 = Blueprint("v1", __name__, url_prefix="/api/v1")
+
+
+def objs_to_dicts_jsonify(obj_list: Query) -> Response:
+    return jsonify([o.to_dict() for o in obj_list])
+
+
+@v1.route("/departments/<int:department_id>/assignments", methods=[HTTPMethod.GET])
+@limiter.limit("5/minute")
+def get_dept_assignments(department_id: int) -> Response:
+    assignments = Department.get_assignments(department_id)
+    return objs_to_dicts_jsonify(assignments)
+
+
+@v1.route("/departments/<int:department_id>/descriptions", methods=[HTTPMethod.GET])
+@limiter.limit("5/minute")
+def get_dept_descriptions(department_id: int) -> Response:
+    descriptions = Department.get_descriptions(department_id)
+    return objs_to_dicts_jsonify(descriptions)
```

Comment:
It looks like the Description contains the creator id and creation time (This is also happening with the CSV exports). Do we have any reason to vend those?

Reply:
Nah, we shouldn't be putting those out.